test/unit/test_diff_filter.py

```diff
@@ -0,0 +1,51 @@
+#!/usr/bin/env python3
+"""
+    Author: Bryan Gillespie
```

Comment:
You can change the comment blocks of the unit tests you wrote to reflect that you are the author (if you wish). I don't want to take credit for someone else's work, though the commit history will show you as the true author.

Reply:
Thanks. Basically, it's not important to me, but maybe someone will have some questions here or someone will find this code awful :) So I'll change it